PlayerCharacterController.cs

```diff
@@ -45,6 +45,8 @@ public class PlayerCharacterController : MonoBehaviour
     [Header("Jump")]
     [Tooltip("Force applied upward when jumping")]
     public float jumpForce = 9f;
+    [Tooltip("Max number of air jumps")]
+    public int maxMultiJumps = 0;
```

Comment:
Why is this 0

Reply:
That's the default value that will be in the unity inspector. While in the unity editor you can increment this value to configure the amount of multijumps to give the player.